client/src/pages/Lobby/Lobby.tsx

```diff
@@ -23,21 +28,89 @@ interface Prop {
 
 //Have a button to start game (logic doesn't need to be implemented yet)
 
-const Lobby: React.FC<Prop> = ({socket, lobbyId, userId }) => {
- const LeaveRoom = (e: FormEvent<HTMLButtonElement>) => {
+export const Lobby: React.FC<ILobbyProps> = ({socket, lobbyId, userId, setMembers, members }) => {
+ 
+ const [notifications, setNotifications] = useState<string[]>([]);
+ const navigate = useNavigate();
+ 
+ 
+ const leaveRoom = (e: MouseEvent<HTMLButtonElement>): void => {
   e.preventDefault();
   if (lobbyId && userId) {
-   socket.emit("leave_room", { lobbyId, userId });
+   socket.emit("leave_room", { lobbyId, userId }, (response: {success: boolean}) => {
+    if(response.success){
+        navigate("/");
+    }
+   });
   }
  };
 
+ const startGame = (e: MouseEvent<HTMLButtonElement>): void => {
+  e.preventDefault();
+  if (lobbyId && userId) {
+   socket.emit("start_game", { lobbyId, userId }, (response: {success: boolean}) => {
+    if(response.success){
+        navigate(`/game:${lobbyId}`);
```

Comment:
Not sure, but maybe wouldn't use a callback for this. I think it's possible that only the player who clicks the button would be navigated to the Game page. So maybe need to change it to have an .on() event handler in App.tsx. Not certain until we create the Game page to test it out.